shared/src/main/scala/scodec/codecs/ChecksumAppendCodec.scala

```diff
@@ -0,0 +1,29 @@
+package scodec
+package codecs
+
+import scodec.bits.BitVector
+
+private[codecs] final class ChecksumAppendCodec[V](
+    target: Codec[V],
+    bitSize: Int,
```

Comment:
bitSize is redundant here

Reply:
No, it is required for undefined size `Codec`s. The requirement is checked in "roundtrip undefined size".